doc/source/developing_models.rst

```diff
@@ -422,7 +438,7 @@ of the 1DPAMZT model is shown below:
     
     
     def main():
-        args = get_options("dpa")
+        args = get_options()
```

Comment:
Just want to confirm that the dropping of "dpa" is intentional

Reply:
That's correct